experimental/packages/api-events/src/types/Event.ts

```diff
@@ -16,7 +16,7 @@
 
 import { Attributes } from '@opentelemetry/api';
 
-export interface LogEvent {
+export interface Event {
```

Comment:
Just to confirm, the field `domain` is optional here only to be able to pass an `Event` object to the EventEmitter. The EventEmitter however will set the domain before emitting the Event further to the processing pipeline.

Reply:
This is the version of the API from a few months ago when we did allow the domain to be overwritten. I will remove it from here.